sympy/core/relational.py

```diff
@@ -171,6 +171,10 @@ def equals(self, other, failing_expression=False):
                 return l
 
     def _eval_simplify(self, ratio, measure):
+        from sympy.simplify import simplify
```

Comment:
You can delete these imports as you haven't added any of them to the code below.